app.py

```diff
@@ -124,10 +124,9 @@ def inject_global_constants():
             ),
             accessibility_statement_url=Config.APPLICANT_FRONTEND_ACCESSIBILITY_STATEMENT_URL,  # noqa
             cookie_policy_url=Config.APPLICANT_FRONTEND_COOKIE_POLICY_URL,
-            contact_us_url=Config.APPLICANT_FRONTEND_CONTACT_US_URL.format(
-                fund_short_name=request.args.get("fund"),
-                round_short_name=request.args.get("round"),
-            ),
+            contact_us_url=Config.APPLICANT_FRONTEND_CONTACT_US_URL
+            + f"""?fund={
+                request.args.get("fund", "")}&round={request.args.get("round", "")}""",
```

Comment:
Was going to suggest using `urlencode` as I seen it on authenticator recently:
https://github.com/communitiesuk/funding-service-design-authenticator/blob/9ccbfefa9511a76f24aae85f7144bdcec2944f5e/api/sso/routes.py#L48

But to be honest, I think the f-strings is the same and more intuitively readable (imo) - maybe we could use a combination of `"` with nested `'` instead of `"""` & `"` though, to get rid of that extra line, but this is very pedantic so feel free to ignore, they're the same anyways